### ISSUE TITLE
Predictions appearing in CSS

### ISSUE THREAD

**Issue by manuman94** [dons20/UIKit-VSCode#16]
Make sure these boxes are checked before (or after) submitting your issue. Thank you!

- [X ] Check that you are running the latest version of Visual Studio Code
- [ X] Check that you are running the latest version of the snippet plugin (UIkit 3.0 Snippets **1.4.0**)
- [- ] Check that you are using a supported file format for the snippets to work in (.html, .php, .js)

### Problem Encountered

Hi, I'm using this extension and it's pretty nice. It makes my developments fastest. So thank you so much @smallp .

The only thing I don't like is that this extension is showing me HTML predictions in CSS files, what doesn't make sense. 

How can I solve this?

### Steps to Reproduce

1. Create a new CSS file and write a word used by this extension to show predictions like "border". Here is an example of what I see:
https://i.imgur.com/p8T7VXc.png

Thank you, greetings.

**Comment by dons20** [comment]
Hi @manuman94, previously I let the snippets only work in certain file environments, but due to the number of requests I got for new file-type additions, I set it so that the snippets would activate anywhere, and would have to be disabled on a workspace/file-by-file basis.

The best suggestion to control where and when certain Intellisense suggestions pop-up is to hit: <br><kbd>F1</kbd> :arrow_right: `Preferences: Configure Language Specific Settings` :arrow_right: Choose `css`.

Then you should be able to set the following: 

```json
"[css]": {
    "editor.wordBasedSuggestions": false
}
```

It's not a perfect solution, but it does prevent the suggestions from popping up while you're working in your css files. There isn't currently any way I've found of disabling a particular set of snippets in specific files. Let me know if this helps.

**Comment by smallp** [comment]
@manuman94 Have you mentioned wrong person?

**Comment by manuman94** [comment]
Yes! I was looking smallp CodeIgniter VSCode extension at the same time and probably I pasted the wrong alias. Sorry for both! (CodeIgniter @smallp extension is also an awesome extension).

Regarding your comment @dons20 , I have other plugins that predict CSS rules and their possible values. Certainly is not essential as I know them, but it makes me write less by autocompleting my words. It would be nice to enable/disable some extensions depending on the file extension, but maybe it's an issue that VSCode developers must see directly.

Greetings!
